dallinger/experiment_server/dashboard.py

```diff
@@ -209,7 +210,26 @@ def heroku_children():
     }
 
     for pane_id, pane in details.items():
-        yield DashboardTab(pane["title"], "dashboard.heroku", None, {"type": pane_id})
+        yield DashboardTab(
+            pane["title"], "dashboard.heroku", None, {"model_type": pane_id}
+        )
+
```

Comment:
Do we have different versions of `black` running? I thought I'd pinned it...

Reply:
19.10b0 which is the pin AFAICT. Is that not black approved Python?

Reply:
I was wondering why `black` seems to have made a change here. Maybe it just wants to feel needed sometimes.